sonar-core/src/main/resources/org/sonar/core/persistence/IsAliveMapper.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="org.sonar.core.persistence.IsAliveMapper">
+
+  <select id="isAlive" resultType="int" >
+    select 1 from schema_migrations where 1 = 1 limit 1
```

Comment:
Unit test is missing to verify that this request is compatible with all db vendors.